app/controllers/admin/users_controller.rb

```diff
@@ -7,12 +7,15 @@
 
 class Admin::UsersController < ApplicationController
 
-  before_action :redirect_if_ldap_user, only: %i[update edit]
-  before_action :authorize_user_class, only: %i[index new create]
-  before_action :authorize_user, only: %i[update unlock edit]
+  before_action :redirect_if_ldap_user, only: [:update, :edit]
+  before_action :authorize_user_class, only: [:index, :new, :create]
+  before_action :authorize_user, only: [:update, :unlock, :edit]
 
   helper_method :update_role
 
+  def edit
```

Comment:
The cop for this is disabled.
`# This cop does not consider action inheritance, therefore disabled Rails/LexicallyScopedActionFilter: Enabled: false`